src/system/sys_profiler.cc

```diff
@@ -133,8 +132,7 @@ void Timer::start() {
 
 
 void Timer::update() {
-    TimeData time = get_time();
-    cumul_time += time - start_time;
+    cumul_time += (TimePoint() - start_time);
```

Comment:
Rather use so simple code diractly since it is better then function named "update" where is not clear what it updates.